### PR TITLE
Telemetry: metric naming standards doc

### PR DESCRIPTION
**Pull request by NajmudheenCT**
This is the guide for developers who develop Adapters,exportes,APIs for metric handling.
OpenSDS has different sources to generate metrics for prometheus.Sources are in the form of exporters or adapters. Here we are defining a standerd of forming metric name an label from the metricpec structure.

### CODE REVIEW

**Comment by NajmudheenCT** [comment]
@xing-yang @leonwanghui @skdwriting @rhsakarpos

File: specs/capri/Metric_Naming_Standards.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# OpenSDS metric Naming standards  
+OpenSDS has different sources to generate metrics for Prometheus. Sources are in the form of exporters or adapters. Here we are defining a standard of forming metric name and labels from the metricSpec structure.
+### metric spec structure 
+model.MetricSpec{
+			InstanceID,
+			InstanceName,
+			Job",
```
Comment (xing-yang) [inline specs/capri/Metric_Naming_Standards.md]: Why is there a " after Job?

File: specs/capri/Metric_Naming_Standards.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+			Associator,
+			Source",
```
Comment (xing-yang) [inline specs/capri/Metric_Naming_Standards.md]: Why is there a " after Source?

File: specs/capri/Metric_Naming_Standards.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# OpenSDS metric Naming standards  
+OpenSDS has different sources to generate metrics for Prometheus. Sources are in the form of exporters or adapters. Here we are defining a standard of forming metric name and labels from the metricSpec structure.
```
Comment (xing-yang) [inline specs/capri/Metric_Naming_Standards.md]: Add new line

File: specs/capri/Metric_Naming_Standards.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+### metric spec structure
```
Comment (xing-yang) [inline specs/capri/Metric_Naming_Standards.md]: New line

File: specs/capri/Metric_Naming_Standards.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+			Unit,
+			IsAggregated,
+			MetricValues[]{timestamp,value},
+		}
+### metric name format
+MetricSpec.Job_MetricSpec.Component_MetricSpec.Name_MetricSpec.unit
+Example:OpensSDS_Volume_ResponseTime_ms
+### metric label format
+  {"instanceID":MetricSpecInstanceID.,"device":MetricSpec.InstanceName,"source":MetricSpec.Source}
+Example: {device="sda",instanceID="sda",source="Node"}
```
Comment (xing-yang) [inline specs/capri/Metric_Naming_Standards.md]: Please add a new line between two sections and also between the title and the content.

**Comment by NajmudheenCT** [comment]
@xing-yang Thanks,
I have fixed all comments.

**Comment by NajmudheenCT** [comment]
@xing-yang Some changes in the metric spec and naming after discussion with Prakash and C2C

**Pull request merged**